tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Console Audit Logging
+Summary: Learn about how to audit operations on the TiDB Cloud Console.
+---
+
+# Console Audit Logging
+
+TiDB Cloud provides you with the audit log feature for Console operations, which can record the history of user access details (such as user login to the Console and cluster creations).
+
+> **Note:**
+>
+> - Currently, the **audit logging** feature is experimental. The interface and output are subject to change.
+> - Console audit logging is enabled by default and the log is not accessible to users without the help of PingCAP support.
+
+To assess whether user access policies and other information security measures of your organization are effective or not, it is a security best practice to periodically analyze the console audit logs.
```

Comment:
There is no problem with the strategy of the audit, which is implemented in stages.
1. At the current stage, users seeking PingCAP support can view audit information.
2. The next stage will enable users to directly access audit log information.
3. If it has to be said that the user cannot view the audit log independently, then the current document may not be provided at all, because the user cannot configure it, and the user cannot view it independently.